purchase_packaging_report/report/purchase_order_templates.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="report_purchaseorder_document"
+        inherit_id="purchase.report_purchaseorder_document"
+    >
+        <xpath expr="//td/span[@t-field='line.product_qty']/.." position="inside">
+            <div
+                t-if="line.product_packaging_id"
+                groups="product.group_stock_packaging"
+                class="text-muted d-inline-block"
+            >
+                <span>(</span>
+                <span t-field="line.product_packaging_qty" />
```

Comment:
In module like this but for sales, fields are shown in different order https://github.com/OCA/sale-reporting/blob/16.0/sale_packaging_report/views/report_sale_order.xml.  
I'm not meaning this is a bad order, but it would be nice if order of fields was the same in both modules.

Reply:
Done